tests/pyfunc/test_model_export_with_loader_module_and_data_path.py

```diff
@@ -267,6 +267,49 @@ def predict(pdf):
     res = pyfunc_model.predict(pdf)
     assert res.dtypes.to_dict() == expected_types
 
+    # 8. np.ndarrays can be converted to dataframe but have no columns
+    with pytest.raises(MlflowException) as ex:
```

Comment:
Can we also test that his works if the schema has no column names?

Reply:
i think this case is tested [here](https://github.com/mlflow/mlflow/pull/3808/files/eaa1adc24505080fcb35a82c7508a3cb9b1d5966#diff-d2db4c32cb1efe50d5cb6196b5e24a807707c07e5806ac8e60c569d6ca3ce8dfR360) under `test_schema_enforcement_no_col_names`